.github/workflows/nightly.yaml

```
@@ -0,0 +1,76 @@
+name: Nightly
+on:
+  schedule:
+    # Times in UTC (PST+8), this translates to running Mon-Fri at midnight/1am (depending on DST)
+    - cron: 0 8 * * 1-5
+  pull_request: {}
```

```suggestion
  pull_request:
    paths: [.github/workflows/nightly.yaml]
```

looked it up after, looks like you can apply the filter on the trigger itself